br-webui/test/test-query-screen.js

```diff
@@ -0,0 +1,28 @@
+var express = require('express');
+var app = express();
+var path = require("path");
+const child_process = require('child_process');
+
+var COMPANION_DIR = "../.."
```

Comment:
should we use `_companion_directory` or `COMPANION_DIR` ?
There is two different variable types for the same thing.

Reply:
They are two independent programs with their own scopes. I can change it if you prefer.

Reply:
They are under the same project, they theoretically should follow the same style/standard names, mainly when they are the same variable.

Reply:
But they aren't the same variable. They are independent contexts.

Reply:
done